www/docs/client/react/usePrefetchQuery.md

```diff
@@ -0,0 +1,83 @@
+---
```

Comment:
in general this page probably needs a short explanation on where and how to use this, it doesn't have to repeat a bunch of stuff from the `useQuery` docs?

Maybe just say like:

`unlike [useQuery](...), `usePrefetchQuery` does not return data ... It is useful when using Suspense in SPAs to fire off a query before a suspense boundary with a children `useSuspenseQuery`... Read more at Tanstack's docs...`